src/unstract/sdk/utils/file_storage_utils.py

```diff
@@ -0,0 +1,61 @@
+import logging
+
+from unstract.sdk.file_storage.fs_impl import FileStorage
+
+logger = logging.getLogger(__name__)
+
+
+class FileStorageUtils:
+    @staticmethod
+    def copy_file_to_destination(
```

Comment:
Should we move this to file_storage.py itself? Make it a FileStorage API....

Reply:
It can use multiple fs insatnce. The FileStorage functions are instance or class properties.